keras_nlp/src/models/deeplab_v3_plus/deeplab_v3_plus_test.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2024 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import pytest
+
+from keras_nlp.src.models.deeplab_v3_plus.deeplab_v3_plus_segmenter import (
+    DeepLabV3Plus,
+)
+from keras_nlp.src.models.resnet.resnet_backbone import ResNetBackbone
+from keras_nlp.src.tests.test_case import TestCase
+
+
+class DeepLabV3PlusTest(TestCase):
+    def setUp(self):
+        self.init_kwargs = {
+            "backbone": ResNetBackbone.from_preset(
+                "hf://timm/resnet18.a1_in1k"
```

Comment:
We want to be able to run most of our testing without a network connect (or slow downloads). Rewrite any testing to create a small test model manually instead of hitting huggingface hub.

Any testing that does require network need to be annotated with `large`, but that should generally be additional testing, not basic functionality testing. See existing backbone/task tests.

Reply:
Added config from the Backbone class for sample